pkg/codegen/testing/test/testdata/simple-enum-schema/nodejs/package.json

```diff
@@ -12,7 +12,7 @@
         "typescript": "^3.7.0"
     },
     "pulumi": {
-        "version": "0.0.1",
-        "resource": true
+        "resource": true,
+        "version": "0.0.1"
     }
 }
```

Comment:
Yup. Since it doesn't change the edited json value I'm ignoring it. Maintaining two different structs to hold the same data is what got us into trouble in the first place, so I don't want to do that to preserve ordering.